### ISSUE TITLE
Error when testing the cityscape model

### ISSUE THREAD

**Issue by ghost** [aimerykong/Recurrent-Scene-Parsing-with-Perspective-Understanding-in-the-loop#1]
Hi aimerykong:

Thanks a lot for releasing this interesting code.  When I tried to test the cityscape model with the released code, the following error comes out:

Error using dagnn.Layer/load (line 200)
No property `usingGlobal` for a layer of type `dagnn.BatchNorm`.

However, such an error does not occur for NYUV2 model. I am wondering is there issues such mismatch between the model file and the code?

**Comment by aimerykong** [comment]
Hi, Dang,

Thank you for pointing out this issue. You are right, I had been making
changes to matconvnet when training the models on different datasets. So
this incompatibility issues arises. Now I fix it and update in github.
Basically I remove the "usingGlobal" field and add new variables.

**Comment by ghost** [comment]
Hi, aimerykong:

Thanks for your fast response! I can confirm this issue has been resolved. And thanks again for sharing.
